api/scpca_portal/management/commands/load_mock_data.py

```diff
@@ -0,0 +1,412 @@
+        "Pons/Cerebellum",
+        "Left cerebral hemisphere",
+        "Left frontal (with satellite lesions in same lobe)",
+        "Cerebellum (multiple masses)",
+        "Right frontotemporal",
+        "Right parietal",
+        "NA",
+        "Right cerebellum",
+        "Left temporal/occipital",
+        "Left frontal",
+        "Right cerebral hemisphere (centered in right parietal lobe)",
+        "C/T-spine",
+        "Pons and surrounding areas",
+        "Left thalamus",
+        "Left temporal",
+        "3rd/lateral ventricles, pineal",
+        "Right frontal leptomeninges",
+        "Posterior fossa",
+        "Right superior temporal gyrus",
+        "Medial left temporal lobe",
+        "Left posterior fossa",
+        "Supracellar",
+        "Thalamus",
+        "Spinal",
+        "Right mesial temporal lobe",
+        "Thalamic",
+        "Basal ganglia",
+        "Right precentral gyrus",
+        "Right parahippocampus",
+        "Cerebellar",
+        "Right temporal",
+        "Left mesial temporal",
+        "Adrenal gland",
+        "Mediastinal mass",
+        "Retroperitoneal mass",
+        "Abdominal mass",
+        "Retrohepatic, perivascular mass, IVC",
+        "Paraspinal mass",
+        "Pelvic mass",
+        "Peritoneal mass",
+        "Aortocaval mass",
+        "Kidney",
+        "Liver",
+        "Right kidney",
+        "Left kidney",
+        "Diaphram",
+        "Lung",
+        "Abdominal wall",
+        "Infratemporal fossa",
+        "Prostate",
+        "Prostate/bladder",
+        "Pelvis",
+        "Stomach",
+        "Thigh",
+        "Parapharyngeal",
+        "Abdomen/pelvis",
+        "Calf",
+        "Chest",
+        "Omentum",
+        "Oropharynx",
+        "Retroperitoneum",
+        "Peritracheal",
+        "Pancreas",
+        "Breast",
+        "Back soft tissue",
+        "Bladder",
+    ],
+    "treatment": [
+        "No treatment",
+        "Treated",
+        "no treatment",
+        "Yes",
+        "No",
+        "Debulking, RT, irinotecan/cetuximab",
+        "GTR, vemurafenib, chloroquine",
+        "STR, Carbo/Vbl",
+        "Upfront resection",
+        "Resection post chemotherapy",
+    ],
+    "sex": ["M", "F", "NA"],
+}
+
+mock_metadata_keys = [
+    "primary_site",
+    "IGSS_stage",
+    "COG_risk",
+    "MYCN_status",
+    "fusion_status",
+    "sample_type",
+    "stage",
+    "group",
+    "risk_stratum",
+    "primary_or_metastasis",
+    "spinal_leptomeningeal_mets",
+    "molecular_characteristics",
+    "outcome",
+    "BRAF_status",
+    "relapse_status",
+    "vital_status",
+    "metastasis ",
+]
+
+mock_sample_index = 0
+
+
+def get_mock_option(key: str, k=False):
+    if k:
+        return random.choices(mock_sample_options[key], k=k)
+    return random.choice(mock_sample_options[key])
+
+
+def purge_all_projects():
+    for project in Project.objects.all():
+        purge_project(project.scpca_id, False)
```

Comment:
This should be True so we purge the old projects data so it doesn't need to be done manually.